AppCenter/AppCenter/Internals/Model/CommonSchema/MSCommonSchemaLog.h

```diff
@@ -0,0 +1,68 @@
+#import "MSAbstractLogInternal.h"
+#import "MSProtocolExtension.h"
+#import "MSUserExtension.h"
+#import "MSOSExtension.h"
+#import "MSAppExtension.h"
+#import "MSNetExtension.h"
+#import "MSLocExtension.h"
+#import "MSSDKExtension.h"
+#import "MSCSData.h"
+#import "MSSerializableObject.h"
+#import "MSModel.h"
+#import "MSCSExtensions.h"
+
+/**
+ * Common schema has one event type with extensions, everything is called an event.
+ */
+@interface MSCommonSchemaLog : MSAbstractLog <MSSerializableObject, MSModel>
+
+/**
+ * The version of the schema. The format is a string with major and minor such as 3.0.
+ */
+@property(nonatomic, copy) NSString *ver;
+
+/**
+ * The event name.
+ */
+@property(nonatomic, copy) NSString *name;
+
+/**
+ * The time on the client when this event was created. Format: A date/time in UTC as the Ticks value.
+ */
+@property(nonatomic) int64_t time;
+
+/**
+ * The effective sample rate for this event at the time it was generated by a client.
+ * The valid range is from a minimum value of one out of every 100 million devices which is "0.000001",
+ * all the way up to all devices which is "100". The default value is 100.
+ */
+@property(nonatomic) double popSample;
+
+/**
+ * An identifier used to identify applications or other logical groupings of events.
+ */
+@property(nonatomic, copy) NSString *iKey;
+
+/**
+ * Event Property flags contain a collection of bits that describe how the event should be processed by the Asimov
+ * pipeline.
+ */
+@property(nonatomic) int64_t flags;
+
+/**
+ * Correlation Vector: A single field for tracking partial order of related telemetry events across component
+ * boundaries.
+ */
+@property(nonatomic, copy) NSString *cV;
```

Comment:
Shouldn't it be "cv"? This is another one that we probably consider to use "correlationVector" (not matching with schema but more readable).

Reply:
Android matches property names with variable names.

Reply:
We decided to stick to the model attribute name. I think the same convention was also adopted on Android.

Reply:
Again, this is how the Common Schema 3.0 is defined as cV.  We do have documentation though.